Gems/PhysX/Code/Include/PhysX/Joint/Configuration/PhysXJointConfiguration.h

```diff
@@ -61,6 +61,7 @@ namespace PhysX
         GenericJointFlag m_flags = GenericJointFlag::None;
         float m_forceMax = 1.0f; ///< Max force joint can tolerate before breaking.
         float m_torqueMax = 1.0f; ///< Max torque joint can tolerate before breaking.
+        AZStd::string m_jointname{""};  // joint_name (useful for some ROS2 application)
```

Comment:
[standard] `m_jointname`> `m_jointName`

I also wouldn't mention other gems in here in the comment, please remove the comment, it's pretty self explanatory that it's a joint name.